Dockerfile-devcore-ubuntu-20.04

```diff
@@ -0,0 +1,31 @@
+# vi:ft=dockerfile
+FROM ubuntu:20.04
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+# SYSTEM
+RUN apt-get -qq update
+RUN apt-get -qq dist-upgrade -y
+RUN apt-get -qq autoremove -y
+RUN apt-get -qq install -y apt-utils
+RUN apt-get -qq install -y ca-certificates tzdata
+RUN apt-get -qq install -y curl wget
+RUN apt-get -qq install -y software-properties-common
+RUN apt-get -qq install -y apt-transport-https
+RUN apt-get -qq install -y sudo
+
+# ENVIRONMENT
+RUN apt-get -qq install -y git perl ruby make g++ pkg-config libidn11-dev libwww-perl libz-dev libaio-dev libncurses-dev libexpat-dev libpcre3-dev libperl-dev libpopt-dev libbz2-dev libtest-simple-perl libsocket6-perl libtest-inter-perl libtest-warn-perl libtest-deep-perl debhelper
```

Comment:
make is already part of build-essential, do we need explicit install here?
also these lib* packages were not present in U18, so are we sure all these are removed from other packages in ubuntu 20?

Reply:
I can also use this docker image for compile third party packages (https://github.com/Zimbra/packages/) and lib* packages are required for third party packages.